core/src/main/scala/filodb.core/metadata/Schemas.scala

```diff
@@ -260,6 +261,23 @@ final case class Schemas(part: PartitionSchema,
 
   schemas.values.foreach { s => _schemas(s.schemaHash) = s }
 
+  /**
+    * This is purely a SWAG to be used for query size estimation. Do not rely for other use cases.
+    */
+  val bytesPerSampleSwag: Map[Int, Double] = {
+    schemas.values.map { s  =>
+      val est = s.data.columns.map(_.columnType).map {
+        case ColumnType.LongColumn => 1
+        case ColumnType.IntColumn => 1
+        case ColumnType.TimestampColumn => 0.5
+        case ColumnType.HistogramColumn => 10
```

Comment:
Why is ColumnType.HistogramColumn 10? Also why is ColumnType.TimestampColumn 0.5 when TimestampColumn is Long?

Reply:
The idea is to have an average bytes per sample. Timestamps are consecutive and with regular interval would compress well. Histograms take more space because of the buckets - this estimate is a swag and in reality it would depend on number of buckets. Long does not assume regular interval and hence is higher than timestamps. I would like @velvia to confirm.

Reply:
When we had done the testing in perf for histograms , the bytePerSample came to around 90 bytes for 64 bucket histograms.